public/src/pages/auth/Login/Login.tsx

```diff
@@ -124,10 +124,9 @@ export const Login: FC = () => {
     getOidcClient().then((client) => setOidcClient(client));
   };
 
-  const extractLogoBgColor = (): string | undefined => {
+  const extractLogoBgColor = (): string => {
```

Comment:
I would prefer to keep the signature which is intended to reflect the method's essence. To prevent the NPE, you just need to adjust the template:
```tsx
<AuthLayout logoBgColor={extractLogoBgColor() ?? 'transparent'}>
```